packages/mgt-spfx/src/libraries/mgt/loc/en-us.js

```diff
@@ -0,0 +1,7 @@
+define([], function() {
```

Comment:
Do we need the localization file for the library?

Reply:
There is no UI, so I could look into removing it altogether.

Reply:
Turns out, this file is needed after all. We can remove its contents, but without it, the SPFx toolchain won't build the project.